bundles/mapping/mapmodule/plugin/vectortilelayer/VectorTileLayerPlugin.js

```diff
@@ -8,6 +8,12 @@ import VectorTileModelBuilder from './VectorTileModelBuilder';
 import VectorTileLayer from './VectorTileLayer';
 import styleGenerator from './styleGenerator';
 
+const LAYER_ID = 'oskariId';
```

Comment:
These could be moved to some "constants" file and reused instead of declaring on multiple places.